Makefile

```diff
@@ -12,6 +12,15 @@ cov_dir = $(top_srcdir)/.coverage
 
 nproc = $(shell nproc)
 orig_go_path = $(shell go env GOPATH)
+
+# The -Wno-deprecated-declarations option is required to suppress deprecation
+# warnings in the gotk3 package.
```

Comment:
Can we add a reference to https://github.com/gotk3/gotk3/issues/420
so that when it is finally fixed we can remove this work-around.